code/game/machinery/computer/crew.dm

```diff
@@ -246,6 +246,9 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 
 		// Binary living/dead status
 		if (sensor_mode >= SENSOR_LIVING)
+			if(tracked_living_mob.IsSleeping() && !isdead(tracked_living_mob))
+				entry["life_status"] = "Alive"
+				return
 			entry["life_status"] = !tracked_living_mob.stat
```

Comment:
```suggestion
			entry["life_status"] = (tracked_living_mob.stat != DEAD)
```
This would be the correct fix I believe, yours would return the entire proc early

Reply:
do need to fix that return, thanks for the heads up. will test your suggested fix

Reply:
well i'll be damned